### ISSUE TITLE
Mir cannot start on bionic

### ISSUE THREAD

**Issue by AlanGriffiths** [MirServer/mir#216]
Neither Mir-on-X nor Mir-on-kms can initialize on current bionic:

```
(gdb) r
Starting program: /home/alan/display_server/mir/cmake-build-debug/bin/miral-shell 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
MIR_CLIENT_PLATFORM_PATH=/home/alan/display_server/mir/cmake-build-debug/bin/../lib/client-modules/
MIR_SERVER_PLATFORM_PATH=/home/alan/display_server/mir/cmake-build-debug/bin/../lib/server-modules/
LD_LIBRARY_PATH=/home/alan/display_server/mir/cmake-build-debug/bin/../lib
exec=/home/alan/display_server/mir/cmake-build-debug/bin/miral-shell.bin
process 19293 is executing new program: /home/alan/display_server/mir/cmake-build-debug/bin/miral-shell.bin
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[2018-03-04 09:06:36.375280] mirplatform: Found graphics driver: mir:mesa-kms (version 0.31.0)
[2018-03-04 09:06:36.375378] <DEBUG> eglstream: EGLStream platform is unsupported: Missing required extensions: EGL_EXT_platform_device EGL_EXT_device_base
[2018-03-04 09:06:36.375394] mirplatform: Found graphics driver: mir:eglstream-kms (version 0.31.0)
[2018-03-04 09:06:36.375968] mirplatform: Found graphics driver: mir:mesa-x11 (version 0.31.0)
[2018-03-04 09:06:36.376087] mirplatform: Found graphics driver: mir:stub-graphics (version 0.31.0)
[2018-03-04 09:06:36.376106] mirplatform: Found graphics driver: throw-on-creation (version 0.31.0)
[2018-03-04 09:06:36.393174] mirserver: Starting
[2018-03-04 09:06:36.393336] mircommon: Loading modules from: /home/alan/display_server/mir/cmake-build-debug/bin/../lib/server-modules/
[2018-03-04 09:06:36.393399] mircommon: Loading module: /home/alan/display_server/mir/cmake-build-debug/bin/../lib/server-modules/graphics-mesa-kms.so.13
[2018-03-04 09:06:36.581690] mircommon: Loading module: /home/alan/display_server/mir/cmake-build-debug/bin/../lib/server-modules/graphics-eglstream-kms.so.13
[2018-03-04 09:06:36.675217] mircommon: Loading module: /home/alan/display_server/mir/cmake-build-debug/bin/../lib/server-modules/server-mesa-x11.so.13
[2018-03-04 09:06:36.675273] mircommon: Loading module: /home/alan/display_server/mir/cmake-build-debug/bin/../lib/server-modules/input-evdev.so.7
[2018-03-04 09:06:36.730790] mircommon: Loading module: /home/alan/display_server/mir/cmake-build-debug/bin/../lib/server-modules/graphics-dummy.so
[2018-03-04 09:06:36.800090] mircommon: Loading module: /home/alan/display_server/mir/cmake-build-debug/bin/../lib/server-modules/graphics-throw.so
[2018-03-04 09:06:36.816618] mircommon: Loading module: /home/alan/display_server/mir/cmake-build-debug/bin/../lib/server-modules/input-stub.so
[2018-03-04 09:06:36.847589] mirplatform: Found graphics driver: mir:mesa-kms (version 0.31.0)
[2018-03-04 09:06:36.847652] <DEBUG> eglstream: EGLStream platform is unsupported: Missing required extensions: EGL_EXT_platform_device EGL_EXT_device_base
[2018-03-04 09:06:36.847665] mirplatform: Found graphics driver: mir:eglstream-kms (version 0.31.0)
[2018-03-04 09:06:36.848452] mirplatform: Found graphics driver: mir:mesa-x11 (version 0.31.0)
[2018-03-04 09:06:36.848512] mirplatform: Found graphics driver: mir:stub-graphics (version 0.31.0)
[2018-03-04 09:06:36.848530] mirplatform: Found graphics driver: throw-on-creation (version 0.31.0)
[2018-03-04 09:06:36.859139] mirserver: Selected driver: mir:mesa-x11 (version 0.31.0)
[2018-03-04 09:06:36.875022] x11-display: Screen resolution = 2560x1440
[2018-03-04 09:06:36.885827] x11-display: 1 visual(s) found
[2018-03-04 09:06:36.885907] x11-display: Using the first one :
[2018-03-04 09:06:36.885959] x11-display: ID            :       33
[2018-03-04 09:06:36.885999] x11-display: screen        :       0
[2018-03-04 09:06:36.886105] x11-display: depth         :       24
[2018-03-04 09:06:36.886144] x11-display: red_mask      :       0xFF0000
[2018-03-04 09:06:36.886175] x11-display: green_mask    :       0xFF00
[2018-03-04 09:06:36.886211] x11-display: blue_mask     :       0xFF
[2018-03-04 09:06:36.886244] x11-display: colormap_size :       256
[2018-03-04 09:06:36.886270] x11-display: bits_per_rgb  :       8
[2018-03-04 09:06:36.943655] mirserver: Using software cursor
[New Thread 0x7fffed6bf700 (LWP 19297)]
[2018-03-04 09:06:36.945282] mirserver: Selected input driver: mir:x11-input (version: 0.31.0)
[2018-03-04 09:06:36.956972] mesa-buffer-allocator: Bound WaylandAllocator display
[2018-03-04 09:06:36.957456] mirserver: Mir version 0.31.0
[New Thread 0x7fffecebe700 (LWP 19298)]
[2018-03-04 09:06:36.972915] GLRenderer: EGL vendor: Mesa Project
[2018-03-04 09:06:36.972955] GLRenderer: EGL version: 1.4 (DRI2)
[2018-03-04 09:06:36.972969] GLRenderer: EGL client APIs: OpenGL OpenGL_ES 
[2018-03-04 09:06:36.972981] GLRenderer: GL vendor: Intel Open Source Technology Center
[2018-03-04 09:06:36.972993] GLRenderer: GL renderer: Mesa DRI Intel(R) HD Graphics 5500 (Broadwell GT2) 
[2018-03-04 09:06:36.973001] GLRenderer: GL version: OpenGL ES 3.1 Mesa 18.0.0-rc4
[2018-03-04 09:06:36.973010] GLRenderer: GLSL version: OpenGL ES GLSL ES 3.10
[2018-03-04 09:06:36.973022] GLRenderer: GL max texture size = 16384
[2018-03-04 09:06:36.974690] GLRenderer: GL framebuffer bits: RGBA=8888, depth=0, stencil=0
[New Thread 0x7fffe7fff700 (LWP 19299)]
[New Thread 0x7fffe77fe700 (LWP 19300)]
[New Thread 0x7fffe6ffd700 (LWP 19301)]
[New Thread 0x7fffe67fc700 (LWP 19302)]
[New Thread 0x7fffe5ffb700 (LWP 19303)]
[New Thread 0x7fffe57fa700 (LWP 19304)]
[2018-03-04 09:06:37.033619] mirserver: Initial display configuration:
[2018-03-04 09:06:37.033660] mirserver: * Output 1: unknown connected, used
[2018-03-04 09:06:37.033693] mirserver: . |_ Physical size 14.6" 317x190mm
[2018-03-04 09:06:37.033709] mirserver: . |_ Power is on
[2018-03-04 09:06:37.033723] mirserver: . |_ Current mode 1200x720 60.00Hz
[2018-03-04 09:06:37.033734] mirserver: . |_ Preferred mode 1200x720 60.00Hz
[2018-03-04 09:06:37.033743] mirserver: . |_ Orientation normal
[2018-03-04 09:06:37.033754] mirserver: . |_ Logical size 1200x720
[2018-03-04 09:06:37.033763] mirserver: . |_ Logical position +0+0
[2018-03-04 09:06:37.033772] mirserver: . |_ Scaling factor: 1.00
[New Thread 0x7fffe4ff9700 (LWP 19305)]
[New Thread 0x7fffcffff700 (LWP 19306)]
Server supports 2 of 10 surface pixel formats. Using format: 3

Thread 11 "miral-shell.bin" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7fffcffff700 (LWP 19306)]
0x00007ffff7302b98 in eglGetDisplay () from /usr/lib/x86_64-linux-gnu/libEGL.so.1
(gdb) bt
#0  0x00007ffff7302b98 in eglGetDisplay () at /usr/lib/x86_64-linux-gnu/libEGL.so.1
#1  0x00005555555ea1aa in MirEglApp::MirEglApp(MirConnection*, MirPixelFormat) (this=0x7fffc8000b70, connection=0x55555617a1e0, pixel_format=mir_pixel_format_argb_8888)
    at /home/alan/display_server/mir/examples/miral-shell/spinner/miregl.cpp:133

```

**Comment by AlanGriffiths** [comment]
This is after the fix to https://bugs.launchpad.net/bugs/1751414

**Comment by RAOF** [comment]
Fixed in libglvnd 1.0.0-2ubuntu2
